storybook/decorators/layout.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+const Container = styled.div`
```

Comment:
Over all looks good. How about replacing `Container` with `Flex`.

Reply:
@srinivasdamam We'd need to apply `height`, `width`, and `margin` properties so there'll be some additional amount of styling to be applied regardless (either using styled-components or a plain styles object / inline style object).

There's also an awkward `import Flex from '../../src/Flex/web';` where you're importing from `src` into a directory outside `src`, and also depending on a particular implementation of `Flex` (`Flex/web`). And tightly coupling your testing / development environment (Storybook) to code itself, so a change to `Flex` could affect how you see things in Storybook.

That's my logic, I can change it if you feel it's better to do so, but you'd have to tell me which approach you want me to style the `Flex` component with (styled-components / JS object / inline styles).